session/session.go

```diff
@@ -2748,8 +2748,6 @@ func CreateSessionWithOpt(store kv.Storage, opt *Opt) (Session, error) {
 	}
 	privilege.BindPrivilegeManager(s, pm)
 
-	sessionBindHandle := bindinfo.NewSessionBindHandle(parser.New())
-	s.SetValue(bindinfo.SessionBindInfoKeyType, sessionBindHandle)
```

Comment:
I removed this because I found it's duplicate: `CreateSessionWithOpt` has already called `createSessionWithOpt` to initialize `sessionHandle`.